apollo-router/src/plugins/subscription.rs

```diff
@@ -758,6 +759,7 @@ mod tests {
                         .unwrap(),
                     )
                     .notify(notify.clone())
+                    .license(LicenseState::default())
```

Comment:
I feel like we could simplify the `fake_builder` to optionally require a `license`,  which would avoid all these changes to unrelated plugins.